plugins/tekton/README.md

```diff
@@ -119,10 +144,38 @@ The Tekton plugin helps with visualizing the `PipelineRun` resources available o
    );
    ```
 
-## Development
+### Development
 
-In [Backstage plugin terminology](https://backstage.io/docs/local-dev/cli-build-system#package-roles), the Tekton plugin is a frontend plugin. You can start a live development session from the repository root using the following command:
+In [Backstage plugin terminology](https://backstage.io/docs/local-dev/cli-build-system#package-roles), the Tekton plugin is a front-end plugin. You can start a live development session from the repository root using the following command:
 
 ```console
 yarn workspace @janus-idp/backstage-plugin-tekton run start
 ```
+
+## For users
+
+### Using the Tekton plugin in Backstage
+
+Tekton is a front-end plugin that enables you to view the `PipelineRun` resources.
+
+#### Prerequisites
+
+- Your Backstage application is installed and running.
+- You have installed the Tekton plugin. For installation process, see [Installation](#installation).
+
+#### Procedure
+
+1. Open your Backstage application and select a component from the **Catalog** page.
+1. Go to the **CI/CD** tab.
+
+   The **CI/CD** tab displays the latest `PipelineRun` resources associated to a Kubernetes cluster. The resources include tasks to complete, and when you hover on a task card, you can view the steps to complete that particular task.
```

Comment:
I'd break up the sentence. "...to complete. When you hover the mouse pointer on a task card..."